### PR TITLE
Library Project completion

### PR DESCRIPTION
**Pull request by Zibilyonik**
The Library Project is completed with the listed features:
-New Book Form
-A card display for books
-Delete Button to delete the books
-Read Button to mark an unfinished book as finished

### CODE REVIEW

**Comment by mimipeshy** [comment]
## STATUS : INVALID SUBMISSION 🔴 

Hello there @Zibilyonik and @TSHEPO-CLOUD 👋🏾 great job so far. However, I cannot review your work because your PR is not professional. A Pull Request should have a meaningful title and a brief description as described [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)

That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+## Screenshot of the page
+![Screenshot Profile Page](./assets/librarypage.png)
+
+
+## Build With
+
+- HTML
+- CSS
+- JavaScript
+- SASS
```
Comment (whiz25) [inline README.md]: - Add a section for `Getting Started`. Add explicit commands for cloning, installing packages, and running the app locally.

File: library.js [hunk 2]
```diff
@@ -1,14 +1,77 @@
...
-let tempLibrary
-
-function Book(title, author, pages, read = false) {
- this.title = title,
- this.author = author,
- this.pages = pages,
- this.read = read
+const libarray = [];
+// Object creator
+function Book(title, author, pages, read) {
```
Comment (whiz25) [inline library.js:3]: - Organize your code into modules. Create a folder `src/modules` and move this file there.

File: library.js [hunk 3]
```diff
@@ -1,14 +1,77 @@
...
+  bookcontainer.appendChild(book);
+}
+// Gets the data from the input form to feed into the "PrintBook" function.
+function GetBook() {
+  const title = document.getElementById('BookTitle').value;
+  const author = document.getElementById('BookAuthor').value;
+  const pages = document.getElementById('BookPages').value;
+  const read = document.getElementById('BookRead').checked;
+  const first = new Book(title, author, pages, read);
+  libarray.push(first);
```
Comment (whiz25) [inline library.js:70]: - Separate JS logic from DOM manipulation logic. Create a file `src/dom.js` and move any dom-manipulation logic there.

File: library.js [hunk 4]
```diff
@@ -1,14 +1,77 @@
...
+  printBook(first);
+}
+// Button listeners to trigger certain events
```
Comment (whiz25) [inline library.js]: - Delete all comments from the codebase

File: library.js [hunk 5]
```diff
@@ -1,14 +1,77 @@
...
+  btndelete.addEventListener('click', () => {
+    const container = btndelete.parentElement.parentElement;
+    container.remove();
+    libarray.splice(book.id - 1, 1);
+  });
+  book.appendChild(btncontainer);
+  bookcontainer.appendChild(book);
+}
+// Gets the data from the input form to feed into the "PrintBook" function.
+function GetBook() {
```
Comment (whiz25) [inline library.js:64]: - Modularise your code. First, define a book model in `src/modules/book.js` before doing operations such as this on the book object.
```javascript
function Book(name, author, pages, read) {
    this.name = name;
    this.author = author;
    this.pages = pages;
    this.read = read;
}
```
- Now you can create a dummy list for books
```javascript
let books = [
Book('','','','')
]
```
- Create a function to loop through the list of books and return the list to your dom-manipulation to be rendered in the UI.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this project. 
### Highlights 🌵 🌵 🌵 
- A user can create a book, mark it as read and delete

### Change(s) to be made 🙅🏿 🙅🏿 🙅🏿 
- Remove comments from your code. They make the codebase look ugly
- Update the readme with instructions for getting started with your project
- Separate JS logic from dom-manipulation logic. Check comments.

### Martin Afani
#### Slack: whiz25

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hello team :hand: 
Good job on this project 
Good luck with the next one

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**